src/crawler.ts

```diff
@@ -2068,6 +2079,10 @@ self.__bx_behaviors.selectMainBehavior();
   async initPages(filename: string, title: string) {
     let fh = null;
 
+    if (this.params.dryRun) {
+      return;
+    }
```

Comment:
Given how much useful information there is in the pages.jsonl files, I'm tempted to let the dry runs write these too.

Reply:
Done, added pages to dry run